xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.496AandB.xml

```diff
@@ -115,12 +115,15 @@
       
         <body>
         <div type="edition">
-         <!--The marked up transcription goes here, surrounded by <ab> </ab> elements-->
-         </div>
+           <ab>
+              <lb n="1"/><persName><name type="gentilicium" key="Catellius"><unclear>C</unclear>atelliu<supplied reason="lost">s</supplied></name></persName>
+              <lb n="2"/><name type="cognomen" key="Primio"><hi rend="tall">i</hi>mio</name> p<expan><abbr>an</abbr><ex cert="low">norum</ex></expan><gap reason="lost" extent="unknown" unit="character"/>
+           </ab>
+        </div>
         
         
                 <div type="translation">
-                    <p/>
+                    <p>[---] Catellius Primius [---] years.</p>
```

Comment:
Is his name Primius or Primio? Seems like there's a little disagreement between your transcription/markup and translation.